Fixes and improvments

BornaBiro opened the pull request:
Product image and case is still missing.

BornaBiro committed:
CSV Fixes.

BornaBiro committed:
Added new font, fixed bugs related to csv, fixed -1 working hour if the login file is missing.,

BornaBiro committed:
Many fixes, mostly code structure.

BornaBiro committed:
Added JSON API.

BornaBiro committed:
Small fixes, README.

BornaBiro committed:
Folder structure changed, fixed daily report.

BornaBiro committed:
Forgot to remove debug code...

BornaBiro committed:
One more repo structure change.

BornaBiro committed:
One more CSF file fix.

BornaBiro committed:
Added JSON pretty print on API.

IgorStrangarevic reviewed (approved):
Par komentara u Jiri SOL-2643

BornaBiro committed:
Fixes.

- Fix for missing image name
- Forced to use .png image
- Text change (worker->employee)
- Time getting fixed (multiple tries, 10 with 500ms delay between)
